shared/actions/platform-specific/index.desktop.tsx

```diff
@@ -9,7 +9,7 @@ import path from 'path'
 import {NotifyPopup} from '../../native/notifications'
```

Comment:
I think we need to change `initializeOpenAtLogin` in this file as well

Reply:
It should be fine now, the correct value gets loaded when opening Settings.